<?xml version="1.0"?>
<launch>
    <node pkg="slowlap_planner" type="slowlap_planner" name="plannerNode" output="screen"/>
    <param name="constant_v" value="true"/>
    <param name="v_max" value="15.0"/>
    <param name="v_const" value="1.0"/>
    <param name="max_f_gain" value="3.0"/>
</launch>

